packages/react/src/components/DetailsList/DetailsList.base.tsx

```diff
@@ -1117,12 +1117,62 @@ export class DetailsListBase extends React.Component<IDetailsListProps, IDetails
   }
 
   /** Builds a set of columns based on the given columns mixed with the current overrides. */
-  private _getFixedColumns(newColumns: IColumn[]): IColumn[] {
+  private _getFixedColumns(newColumns: IColumn[], viewportWidth: number): IColumn[] {
+    const { selectionMode = this._selection.mode, checkboxVisibility } = this.props;
+    let remainingWidth = viewportWidth - 32; //32px buffer accounts for a vertical scrollbar and minor layout shifts.
```

Comment:
I would like to avoid having `32px` hard-coded into here. Depending on how `DetailsList` is hosted, this won't be necessary at all, and the width of a scrollbar is both platform-dependent and zoom-dependent. Consider taking this in as a `columnMargin` optional parameter or something.

Reply:
This should be fixed now